src/components/ProjectPreview.jsx

```diff
@@ -11,7 +12,8 @@ export default function ProjectPreview({preview, project}) {
         {preview.slice(0, MAX_LENGTH)}
       </div>
       <div className="project-preview__timestamp">
-        {moment(project.updatedAt).fromNow()}
+        {project.updatedAt ?
+          moment(project.updatedAt).fromNow() : t('top-bar.pristine-project')}
```

Comment:
The “pristine project” terminology is used in the codebase but isn’t really intended to be user-facing—how about we just omit this element altogether in the pristine case?

Reply:
Okay